stable/storages/README.md

```diff
@@ -0,0 +1,62 @@
+# storages
+
+This chart is used to apply pv and pvc. You can specify any number of pv and pvc. This chart can come in handy when a public chart does not offer additional pv and pvc. You have to use the following format to specify pv and pvc:
```

Comment:
README need more context, perhaps a use case of combining this with the “public chart”.

Reply:
Updated with an example